modules/storage-persistent-storage-azure-file-pod.adoc

```diff
@@ -23,8 +23,9 @@ kind: Pod
 metadata:
   name: pod-name <1>
 spec:
+  # ...
   containers:
-    ...
+    # ...
```

Comment:
Here seems a little strange for me.

Reply:
In our guidelines [1], when we want to show that example YAML is truncated, we use `...`, but we comment it out, so that it remains valid YAML. So we use `# ...`

I've moved it to be left-aligned properly though.

[1] https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc

Reply:
Glad to know it.